tensorboard/webapp/header/header_component.scss

```diff
@@ -44,3 +44,7 @@ settings-button {
   height: 100%;
   overflow: hidden;
 }
+
+.readme mat-icon {
+  color: #fff;
```

Comment:
Could you read this from the palette instead?

Reply:
I am trying to stay consistent with the other buttons.

Reply:
I believe I took care of this in #6620.

Reply:
Oh cool. Thanks for catching this!